core/chains/evm/legacy.go

```diff
@@ -121,6 +121,14 @@ func SetupNodes(db *sqlx.DB, cfg LegacyEthNodeConfig, lggr logger.Logger) (err e
 	if err = json.Unmarshal([]byte(cfg.EthereumNodes()), &nodes); err != nil {
 		return errors.Wrapf(err, "invalid EVM_NODES json, got: %q", cfg.EthereumNodes())
 	}
+
+	// Nodes cannot specify 'send_only' and 'ws_url' simultaneously - preempt the eventual persistence failure (due to a constraint)
+	for _, evmNode := range nodes {
+		if evmNode.SendOnly && evmNode.WSURL.Valid {
+			return errors.New("Invalid EVM node configuration:  A node cannot simultaneously specify both 'sendOnly' and 'wsUrl'")
```

Comment:
I'm not sure I understand,  there are error and logger messages, in that file for example, that mix upper and lowercase sentence starts, misuse semicolons, et al.  Do we have a coding standard/style guide somewhere I can use?

Shouldn't "A node" then become "a node"?
Should I fix the other messages while I'm in there?

Most error message standards (e.g. the GNU coding standards) suggest interactive programs (and by extension the interactive aspects of a non-interactive program, such as starting up our node) use capital letters to start messages (but not add periods)

The one space/two space thing isn't important, so happy to use one space.

Reply:
Also, these are important things for me to learn - I appreciate you pointing them out now :)

Reply:
Generally speaking, log messages should be capitalized and errors should not be. IIRC the linter normally enforces the error standard, and will be restored momentarily. 

s/A/a makes sense too - I hesitated on that.

The policy in practice is only worry about new/changed code, but if you want to clean up other outstanding ones that would be welcome.

Reply:
IMHO it helps readability to capitalize log messages, but we don't normally enforce that one as strictly, and some lower case messages may be leftover from a time when a prefix was added: `SystemName: lowercase message` - some output formats may still even do this :shrug:.
I think this kind of work is worthwhile, but maybe as separate PRs. Also, this particular file isn't the best investment, since it is legacy code that will be removed in 2.0.0